sim/priest/mindblast.go

```diff
@@ -43,6 +43,10 @@ func (priest *Priest) newMindBlastTemplate(sim *core.Simulation) core.SimpleSpel
 
 	baseCast.Cooldown -= time.Millisecond * 500 * time.Duration(priest.Talents.ImprovedMindBlast)
 
+	if ItemSetAbsolution.CharacterHasSetBonus(&priest.Character, 4) { // Absolution 4p adds 10% damage
+		effect.StaticDamageMultiplier += 0.1
```

Comment:
Usually multiplers are included multiplicatively. Is there a reason its different here?

Reply:
:facepalm: thank you for catching that